lca-cli/seedcreator/seedcreator.go

```diff
@@ -147,7 +147,13 @@ func (s *SeedCreator) CreateSeedImage() error {
 		return fmt.Errorf("failed to run once backup_mco_config: %w", err)
 	}
 
-	if err := s.createAndPushSeedImage(); err != nil {
+	clusterInfoJSONSBytes, err := os.ReadFile(path.Join(s.backupDir, common.SeedClusterInfoFileName))
+	if err != nil {
+		return fmt.Errorf("failed to read cluster info file: %w", err)
+	}
+
+	clusterInfoJSON := string(clusterInfoJSONSBytes)
+	if err := s.createAndPushSeedImage(clusterInfoJSON); err != nil {
```

Comment:
Do we really want all this in a label? Why not just validate the data from the manifest.json after pulling the image? I get that it would save the pull, but is there some other benefit?

Reply:
That said, I do like the idea of this being an easy way to access the manifest.json data without pulling the image, for expanded image validation. Are there any cons to it? Like would we end up with restrictions on what could get added to the manifest.json? I think no, but I'm not sure

Reply:
Yeah exactly, it's nice to have. Couldn't think of any cons. It's information that's already in the seed, just now duplicated to different location as well (labels on top of inside a file in a layer), so I don't see any additional compatibility restrictions this would add.

>Like would we end up with restrictions on what could get added to the manifest.json? 

Perhaps in terms of size. I guess at that point in the future it would be an obvious problem and we could redact any future huge thing we put in `manifest.json` before we serialize it to put inside the label